action.yml

```
@@ -21,6 +21,14 @@ inputs:
   codecov-token:
     description: >-
       The Codecov token to use when uploading coverage data.
+  controller-docker-opts:
+    description: |
+      Comma-separated controller docker options.
+      seccomp=default|unconfined
+      cgroup=none|v1-only|v2-only|v1-v2
+      audit=none|required
+      privileged=yes|no
+      memory=<integer>
```

I would format them properly:
```suggestion
      Comma-separated controller docker options:

      - `seccomp=default|unconfined`
      - `cgroup=none|v1-only|v2-only|v1-v2`
      - `audit=none|required`
      - `privileged=yes|no`
      - `memory=<integer>`
```